readthedocs/config/tests/test_config.py

```diff
@@ -274,15 +274,15 @@ def test_python_pip_install_default():
     assert build.python.install_with_pip is False
 
 
-def describe_validate_python_extra_requirements():
+class TestValidatePythonExtraRequirements(object):
 
-    def it_defaults_to_list():
+    def test_it_defaults_to_list(self):
```

Comment:
not sure if we need the `it` in the tests names anymore, what do you think?

Reply:
I'm not an english expert, but I guess 2 makes sense.

Reply:
Yea, 2 is 👍